packages/engines/scripts/postinstall.js

```diff
@@ -1,11 +1,10 @@
-const fs = require('fs')
 const path = require('path')
 
 const postInstallScriptPath = path.join(__dirname, '..', 'dist', 'scripts', 'postinstall.js')
 
 // that's when we develop in the monorepo, `dist` does not exist yet
 // so we compile postinstall script and trigger it immediately after
-if (fs.existsSync(postInstallScriptPath) === false) {
+if (require('../package.json').version === '0.0.0') {
```

Comment:
This will ensure that this script can be re-run after `engines-version` updates (they are bundled into the script).

Reply:
@millsp  This line could fail I think, we could add some try catch here to avoid any issue on the users side?

Reply:
Done.